client/src/js/app.js

```diff
@@ -4,7 +4,7 @@ var bhima = angular.module('bhima', [
   'ngStorage', 'chart.js', 'tmh.dynamicLocale', 'ngFileUpload', 'ui.grid',
   'ui.grid.selection', 'ui.grid.autoResize', 'ui.grid.resizeColumns',
   'angularMoment', 'ngMessages', 'ui.grid.pagination', 'ui.grid.moveColumns',
-  'ui.grid.grouping'
+  'ui.grid.grouping', 'growlNotifications', 'ngAnimate'
```

Comment:
You import ngAnimate here, but I don't see the `bower.json` file changed in this PR.  Please add the `ngAnimate` library to our bower dependencies, then add the tag `needs bower install` to this PR.